src/config.erl

```diff
@@ -183,6 +187,26 @@ delete(Section, Key, Persist, Reason) when is_list(Section), is_list(Key) ->
     gen_server:call(?MODULE, {delete, Section, Key, Persist, Reason}).
 
 
+features() ->
+    lists:usort([list_to_binary(Key) || {Key, "true"} <- ?MODULE:get(?FEATURES)]).
+
+
+feature_enable(Feature) when is_atom(Feature) ->
```

Comment:
Yeah enable_feature/disable_feature sounds better. Can make it an atom as well.

vs `add/remove`, `enable/disable` has more of an idempotent feel to it.  But ok with either way.